x/mongo/driver/topology/rtt_monitor.go

```diff
@@ -215,11 +251,7 @@ func (r *rttMonitor) addSample(rtt time.Duration) {
 
 	r.samples[r.offset] = rtt
 	r.offset = (r.offset + 1) % len(r.samples)
```

Comment:
As far as I can tell, the existing logic for calculating "min" and "p90" are only used in the `Stats` method. If that's the case, we should remove the existing logic for calculating "min" and "p90" and update the `Stats` method to return information based on the actually used "min" implementation.